language/repl.md

```diff
@@ -0,0 +1,20 @@
+# REPL
+
+REPL stands for "read eval print loop". It's a convenient tool that
+reads a nix-expression and prints the evaluated result.
+
+Nix has such a REPL:
+
+```
+nix-env -iA nixpkgs.nix-repl
```

Comment:
nvm just saw the stuff below, it should say that the repl exists but needs to be installed by the following command